apis/goharbor.io/v1alpha2/harborcluster_types.go

```diff
@@ -130,6 +131,12 @@ type MinIOSpec struct {
 	// Number of persistent volumes that will be attached per server
 	// +kubebuilder:validation:Minimum:=1
 	VolumesPerServer int32 `json:"volumesPerServer"`
+	// ExternalCertSecret allows a user to specify custom CA certificate, and private key. This is
+	// used for enabling TLS support on MinIO Pods.
+	// +kubebuilder:validation:Optional
+	ExternalCertSecret *minio.LocalCertificateReference `json:"externalCertSecret,omitempty"`
```

Comment:
It seems `ExternalCertSecret` and `Host` should be set only when the` redirect.disable=false`, right? So, it may be better to put these options into the `Redirect StorageRedirectSpec` section as part of it?